Config/supersid.cfg

```diff
@@ -77,4 +77,15 @@ ftp_server = sid-ftp.stanford.edu
 ftp_directory = /incoming/SuperSID/NEW/
 # local_tmp shall be an absolute path or a path relative to the supersid script folder
 local_tmp = ../outgoing
-call_signs = NWC
+call_signs = NWC   
+
+[Email]
+from_mail =  
+email_server =   
+email_port =  
+email_tls =  
+email_login =  
+email_password =   
+
+# email_server for gmail would be smtp.gmail.com  
+# email_port for gmail would be 587  

```

Comment:
This change is already part of my pending pull request including some sample values.